go.mod

```diff
@@ -25,13 +25,13 @@ require (
 	github.com/urfave/cli/v2 v2.25.7
 	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.42.0
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.43.0
-	go.opentelemetry.io/otel v1.17.0
+	go.opentelemetry.io/otel v1.18.0
```

Comment:
I guess this is fine as long as `go.opentelemetry.io/otel/exporters/jaeger v1.17.0` stays the same